src/sql/workbench/browser/modelComponents/button.component.ts

```diff
@@ -159,7 +159,7 @@ export default class ButtonComponent extends ComponentWithIconBase implements IC
 	}
 
 	public get buttonType(): azdata.ButtonType {
-		return this.getPropertyOrDefault<azdata.ButtonProperties, azdata.ButtonType>((props) => props.buttonType, azdata.ButtonType.Normal);
+		return this.getPropertyOrDefault<azdata.ButtonProperties, azdata.ButtonType>((props) => props.buttonType, <azdata.ButtonType>'Normal');
```

Comment:
@anthonydresser Do you know why the import rules don't allow importing from https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/api/common/sqlExtHostTypes.ts in places like this?

We allow the import in `**/{vs,sql}/workbench/contrib/**/common/**` (https://github.com/microsoft/azuredatastudio/blob/main/.eslintrc.json#L701) which includes things like https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/contrib/dashboard/browser/widgets/explorer/explorerWidget.component.ts#L13 so I'm not sure why we wouldn't allow it here too.